integration/test_flow_integration.py

```diff
@@ -760,7 +798,10 @@ def enrich(event, state):
 
 
 def test_write_table_metadata_columns(setup_teardown_test):
-    table = Table(setup_teardown_test.table_name, setup_teardown_test.driver())
+    if setup_teardown_test.driver_name == "SQLDriver":
```

Comment:
apart from the skip of test_flow_aggregation (which is "by design" since you don't support aggregation with this driver), why are other tests  files skipped and only test_flow_integration.py is used?
Also in test_flow_integration.py, 8 tests are skipped with vague "not testing over SQLDriver".
Please see what tests you can add. If there is good/technical reason to skip a test please state it in the message.

Reply:
all the tests in the integration are supported now for this driver beside : 
1. kv tests
2. `test_writing_timedelta_key` - because sqlite issues
3. `test_write_three_keys_to_v3io_from_df` - waiting for your answer
4. `test_write_none_time` - when the time filed is none this test expected not to get the time field when querying the drivers - I don't think it something we want from sql driver.

Reply:
regarding test_write_three_keys_to_v3io_from_df, moshe.dc1e617eaae40eea3449baf3795bb5b50676c963 is the key used to store the first record in the table (user "moshe")
regarding test_write_none_time, why not?